dev_tools/configure_vscode_for_bazel.py

```diff
@@ -89,11 +109,29 @@ def parse_arguments(argv: Sequence[str] | None = None) -> argparse.Namespace:
         help="Additional arguments to pass to the `compile_commands.json` refresh template.",
     )
 
+    add_run_and_debug_arguments(parser)
```

Comment:
```suggestion
    parser = add_run_and_debug_arguments(parser)
```

Although python is referenced based, this seems to be a patter just better visualize that you are modifying something